developers/weaviate/starter-guides/managing-collections/collections-scaling-limits.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Do you have too many collections?
+sidebar_label: Scaling limits with collections
+sidebar_position: 1
+image: og/docs/more-resources.jpg
+# tags: ['performance']
+---
+
+:::info
+Added in `v1.25.34`, `v1.26.18`, `1.27.15`, `1.28.9`, `1.29.2`
+:::
+
+:::caution You hit the collections count limit?
+If you see an error that the collection count limit has been reached, it means you **can't create any more collections** because the number of collections is affecting performance. This limit exists to control indexing, storage, and management complexity.<br /><br />
+**Instead of simply raising the limit, consider rethinking your architecture**. If you really need to change the limit, use the [`MAXIMUM_ALLOWED_COLLECTIONS_COUNT`](/developers/weaviate/config-refs/env-vars) environment variable.
+:::
+
+This guide offers an overview of the available architectural choices of using **multi-tenancy** or defining a **dedicated collection** for each subset of data.
+
+Consider a scenario where you are developing a SaaS platform for product recommendations, allowing users (merchants) to import and vectorize their product data. The users of this platform require the ability to efficiently perform vector searches to generate personalized recommendations for users on their respective e-commerce platforms.
+
+One option is creating a dedicated collection for each user to store their product data. However, as the number of users grows, so does the number of collections, potentially leading to performance bottlenecks and increased operational complexity. This leads us to an important question: **How many collections is too many?**
+
+## Structuring data in a vector database
+
+import MultiTenanyVsMultipleCollections from '/developers/weaviate/starter-guides/img/weaviate-multi-tenancy-vs-multiple-collections.png';
+
+<div class="row">
+  <div class="card">
+    <div class="card__image">
+      <img
+        src={MultiTenanyVsMultipleCollections}
+        alt='Multi-tenancy vs "One collection per user" architecture'
+      />
+    </div>
+  </div>
+</div>
+<br />
+
+When designing a vector database collection definition (data schema), you must decide between **multi-tenancy** (storing data for multiple tenants in a single collection) or creating **separate collections for each data subset** (the "One collection per user" strategy). Each approach has its own advantages and trade-offs, especially in terms of performance, scalability, and management.
```

Comment:
> When designing a vector database collection definition

IMO should be `When designing a Weaviate collection definition` - it's not a universal vector DB thing.